tests/tests.py

```diff
@@ -619,6 +623,27 @@ def test_add_and_remove_user(self):
         result = self.jira.delete_user(self.test_username)
         assert result, True
 
+    def test_create_team(self):
```

Comment:
I would look to create any tests in a new `tests/resources/test_teams.py`

I would look at `test_filter`, the way it uses a context manager may also be helpful for you